### DISCUSSION TITLE
Monkey365 and Latest CIS Benchmarks

### DISCUSSION THREAD

**Discussion by XaZzZX** [silverhack/monkey365#116] [General]
Hi

Based on the description, the following standards are supported by Monkey365:

CIS Microsoft Azure Foundations Benchmark v1.4.0
CIS Microsoft 365 Foundations Benchmark v1.4.0
CIS Microsoft Azure Foundations Benchmark v1.5.0
CIS Microsoft 365 Foundations Benchmark v1.5.0

However, we now have updated versions of CIS Benchmarks for 365 and Azure which is v3.0.0. are the checks of Monkey365 updated according to the new benchmarks?

thanks

**Comment by silverhack** [comment]
Hi @XaZzZX, the latest version for M365 benchmark is [2.0](https://github.com/silverhack/monkey365/blob/main/rules/rulesets/cis_m365_2.0.json). I'll try to update benchmarks as soon as I can. Please also note that PR are welcome in case you want to help. 

Cheers,
